guide/english/certifications/responsive-web-design/basic-css/use-css-selectors-to-style-elements/index.md

```diff
@@ -3,8 +3,14 @@ title: Use CSS Selectors to Style Elements
 ---
 ## Use CSS Selectors to Style Elements
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/basic-css/use-css-selectors-to-style-elements/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+CSS selectors are widely used to apply styling to all instances of a particular element.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+To apply the color `purple` to all the `h2`s on the page, you can define the following CSS inside a `<style></style>` block.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+```css
+h2 {
+    color: purple;
```

Comment:
```suggestion
  color: purple;
```
We use two spaces for each indentation level, so please change this to the recommended one